src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -3913,7 +3916,7 @@ public SendInvalidateRequest(): base("sendInvalidate")
     internal class SendInvalidateArguments : DebugRequestArguments
     {
 
-        public InvalidatedAreas Areas { get; set; }
+        public InvalidatedAreas[]? Areas { get; set; }
```

Comment:
```suggestion
        public List<InvalidatedAreas> Areas { get; set; }
```

Working with a collection is easier than a raw array.